.travis.yml

```diff
@@ -33,16 +32,19 @@ addons:
       - libclang-8-dev
 
 install:
-  - export TAG=v8.0.1-3
-  - export TARBALL=SPIRV-LLVM-Translator-${TAG}-linux-${BUILD_TYPE}.zip
-  - wget https://github.com/KhronosGroup/SPIRV-LLVM-Translator/releases/download/${TAG}/${TARBALL} -O /tmp/${TARBALL}
-  - unzip /tmp/${TARBALL} -d spirv-llvm-translator
+  - git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git -b llvm_release_80 spirv-llvm-translator
+  - cd spirv-llvm-translator
+  - for patch in `ls ../patches/spirv/*.patch`; do echo $patch; git apply $patch; done
```

Comment:
Should we fail the build if we couldn't apply a patch? Probably `set -e` can help with it,

Reply:
Maybe. The build can pass without these patches.
Is it possible to merge this patches to spirv translator, instead of keeping it here and applying it?

I'll merge this pull request firstly. If needed, I'll add "set -e" to stop build if patch cannot be applied.

Thanks.

Reply:
> Is it possible to merge this patches to spirv translator, instead of keeping it here and applying it?

It depends on patches from patches/clang directory. Patches for spirv translator will not work without patches to clang.